examples/ContractCall/ContractCall.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <Nullable>disable</Nullable>
+    </PropertyGroup>
+
+    <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+        <Exec Command="dotnet nccs &quot;$(MSBuildProjectFile)&quot;" />
+    </Target>
+
+    <ItemGroup>
+        <PackageReference Include="Neo.SmartContract.Framework" Version="3.6.0" />
```

Comment:
This PR is great. But it should be included in devpack. Otherwise making `Neo.SmartContract.Framework` to a specific version is not good and not esay to maintain and update. New syscall will surely new framework dependency and break something there if it's also applied on exmaple. If devpack is not well-known, maybe we can move it to neo as well :)

Reply:
In devpack, we can easily make the dependency path as:
```
  <ItemGroup>
    <ProjectReference Include="..\Neo.SmartContract.Framework\Neo.SmartContract.Framework.csproj" />
  </ItemGroup>
```

Reply:
From my view, we should forget about `WHAT IS RIGHT` or `WHAT SHOULD BE`.  Working in a `correct` or `expected` way for us, but inconvenient for developers means nothing. Who will care about a `make sense` solution if they can't event see it.

As to including compiler into the monorepo, i got many suggestions to keep it seperate.

Reply:
> From my view, we should forget about `WHAT IS RIGHT` or `WHAT SHOULD BE`. Working in a `correct` or `expected` way for us, but inconvenient for developers means nothing. Who will care about a `make sense` solution if they can't event see it.
> 

I suggest caring more about the long term, hard code dependency is not good for smart contract examples, it should keep updating. No one knows how framework will be changed in the future, compatible or incompatible.

Reply:
we can update them from time to time, its source code, people will clone source code, build source code. If there are changes to make, we can do it in prs. Why hardcode referecnce can be an issue for examples?

Reply:
Also contracts shouldn't change the blockchain. So @superboyiii that should not be a problem. For example like `bNeo` it can only be compiled with `3.2` or something like that. Doesn't mean you can't deploy it. The assembly for the `VM` doesnt change last i checked. The `opcodes` will always be the same. The **only** big difference is supporting `C#` syntax. So with that said, it doesn't matter which version of the `nccs` you use. The only difference would be new features.

Reply:
> Also contracts shouldn't change the blockchain. So @superboyiii that should not be a problem. For example like `bNeo` it can only be compiled with `3.2` or something like that. Doesn't mean you can't deploy it. The assembly for the `VM` doesnt change last i checked. The `opcodes` will always be the same. The **only** big difference is supporting `C#` syntax. So with that said, it doesn't matter which version of the `nccs` you use. The only difference would be new features.

I don't mean blockchain data incompatible, I mean incompatible code syntax changes, and if we add a new syscall in framework and want it be applied on examples, we need update all these framework to the latest otherwise it can't be compiled.

Reply:
If we add a new syscall without publishing it, why would we expect developers to use it in examples? They can not deploy them anyway.

Reply:
>I don't mean blockchain data incompatible, I mean incompatible code syntax changes, and if we add a new syscall in framework and want it be applied on examples, we need update all these framework to the latest otherwise it can't be compiled.

`Directory.Build.props` file would of been needed than.